### DISCUSSION TITLE
¿Piensas que es necesario que Searchpe ofresca authenticación/autorización en sus endpoints?

### DISCUSSION THREAD

**Discussion by carlosthe19916** [project-openubl/searchpe#82] [General]
Actualmente Searchpe ofrece todos sus endpoints a travéz de REST endpoints que no ofrecen authenticación/autorización; es decir, todo aquel que tenga accesso al host/port del servicio podrá acceder a Searchpe.

¿Piensas que es necesario que Searchpe ofresca authenticación/autorización en sus endpoints?

Déjanos tus comentarios y requerimientos aquí.

**Comment by jvargada** [comment]
Sería fabuloso para poder controlar a quienes damos acceso al servicio y en que momento denegarlo.

**Reply by carlosthe19916** [reply]
Gracias por el feedback, este requerimiento es solucionado por https://github.com/project-openubl/searchpe/pull/103 y estará disponible en la siguiente version de Searchpe.
